libraries/Microsoft.Bot.Builder.AI.QnA/Dialogs/QnAMakerDialog.cs

```diff
@@ -290,6 +290,24 @@ public QnAMakerDialog([CallerFilePath] string sourceFilePath = "", [CallerLineNu
         [JsonProperty("rankerType")]
         public StringExpression RankerType { get; set; } = new StringExpression(RankerTypes.DefaultRankerType);
 
+        /// <summary>
+        /// Gets or sets a value indicating whether gets or sets enablePreciseAnswer. 
```

Comment:
Grammar needs copyediting.

Reply:
Fixed.

---
In reply to: [430548228](https://github.com/microsoft/botbuilder-dotnet/pull/3935#discussion_r430548228) [](ancestors = 430548228)